manifests/configmap.ee.yaml

```diff
@@ -0,0 +1,15 @@
+---
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: backend-config
+  namespace: :env:
+  labels:
+    app: backend
+    environment: :env:
+data:
+  AWS_REGION: eu-west-1
+  AUTH_MAGIC_LINK_ENABLED: "false"
+  AUTH_ESTONIAN_ID_ENABLED: "true"
```

Comment:
Could we also add `DEFAULT_LANGUAGE_CODE` to both?

Reply:
(`"et"` and `"en"`)

Reply:
We can add it ourselves in another pr